src/include/openenclave/internal/utils.h

```diff
@@ -0,0 +1,233 @@
+// Copyright (c) Open Enclave SDK contributors.
```

Comment:
If we can avoid internal OE headers, we should. I can't easily see why we need it here. If it's just for the round-up to the next power of 2, I think we should re-implement that ourselves.

Reply:
Done.